system_tests/local/ec2_test_utils.py

```diff
@@ -224,3 +284,9 @@ def _create_instance(self, ec2_client):
             image_id=self.env.ubuntu_trusty_image_id,
             instance_type=self.env.micro_instance_type)
         return new_reservation.instances[0]
+
+    def _create_external_load_balancer(self):
+        return self._get_elb_client().create_load_balancer(
```

Comment:
it's not a big deal, because in almost all cases the outcome will be the same. but since in the def directly above it, we pass the client as an argument, instead of calling a method from a private function, i think it's better.